release.yaml

```diff
@@ -56,7 +56,9 @@ rancher-provisioning-capi:
   - 105.0.0+up0.4.0
 rancher-vsphere-cpi:
   - 104.0.1+up1.8.1
```

Comment:
We probably want to remove 104.x.x charts from here - looks like they were added by https://github.com/rancher/charts/pull/4559 and I'm not sure it was intentional.
@nicholasSUSE , could you please I'm not missing a reason why we need those here.

Reply:
This is expected per https://github.com/rancher/charts/pull/4634#pullrequestreview-2393557550